unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1819,13 +1819,12 @@ handleUpdate input maybePatchPath hqs = do
             b <- getAt p
             eval . Eval $ Branch.getPatch seg (Branch.head b)
       let patchPath = fromMaybe defaultPatchPath maybePatchPath
-      slurpCheckNames <- slurpResultNames
```

Comment:
https://github.com/unisonweb/unison/blob/ac9da6faec2ff5fd32ad3f8935ae342ecdfb7b74/unison-cli/src/Unison/Codebase/Editor/HandleInput.hs#L3329-L3332

shows that slurpCheckNames is exactly currentPathNames.

If anyone has a preference for which name we use where just let me know 👍🏼